angular/projects/admin-nrpti/src/app/services/factory.service.ts

```diff
@@ -350,6 +350,139 @@ export class FactoryService {
     return this.recordService.deleteRecord(collectionId, 'collection');
   }
 
+  /**
+   * Edit a mine record.
+   *
+   * @param {*} record object containing mine record values to update into a record item.
+   * @returns {Observable<object>}
+   * @memberof FactoryService
+   */
+  public editMineRecord(record: any): Observable<object> {
+    const outboundObject = {};
+
+    // theres a bunch missing?
```

Comment:
Are there?

Reply:
Well, there was! That comment doesn't make as much sense anymore.

However, does bring up a note that some of the types in the list do not have BCMI flavours and therefore will not work. Those types should either have flavours added, or be removed from the list if they are not needed for BCMI.